SecretSanta/src/SecretSanta.Business/Gift.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Business
+{
+    public class Gift
+    {
+        private int _Id;
+        private string _Title;
+        private string _Description;
+        private string _Url;
+        private User _User;
+
+#pragma warning disable CA1054 // Uri parameters should not be strings
+        public Gift(int Id, string Title, string Description, string Url, User User)
+#pragma warning restore CA1054 // Uri parameters should not be strings comment: Allowing to be to spec of the assigment
+        {
+            this._Id = Id;
```

Comment:
```suggestion
            Id = id;
```
Change the private field _Id to a property called Id (use PascelCase for naming) and pass your parameters with the camelCase naming convention. 

The this operation is only needed if you have two variables that are the same name, it helps distinguish which is the paramater and which isn't.